plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -947,6 +947,7 @@ read_only::get_info_results read_only::get_info(const read_only::get_info_params
       //std::bitset<64>(db.get_dynamic_global_properties().recent_slots_filled).to_string(),
       //__builtin_popcountll(db.get_dynamic_global_properties().recent_slots_filled) / 64.0,
       app().version_string(),
+      symbol().name(),
```

Comment:
To successfully interact with the chain, you also need to know the precision of that symbol, which `symbol().name()` does not provide.  I'm not certain what the best format of that string should be.  Perhaps @jcalfee can help.  At the very least, it should be `precision,name`.  It may be even better to provide precision in another field.

Reply:
Thank you so much for your prompt review of this. The use case that this is valuable for is to support EOSIO-based blockchains that has non EOS prefixes. The precision is irrelevant in these cases, as the prefix is the only factor when generating the public key. Please see the following in eosjs-ecc for reference. https://github.com/EOSIO/eosjs-ecc/pull/30

The general idea is this:

1) User downloads an EOSIO tool (e.g. wallet, toolkit) that involves some sort of key-creation/key lookup process.

2) Tool asks user for an API endpoint

3) With a simple call to get_info for the specified API endpoint, the tool gets the chainId, and the CORE_SYMBOL associated with said chain

4) For the tool's "Generate Key"/"Validate Key"/"Lookup Account" features, it knows which prefix is being used by that chain based on the results from get_info. As a result, it's able to call the functions in eosjs and eosjs-ecc [such as eosjs-ecc. privateToPublic("key", "key_prefix")] automatically w/o any additional interfaces requesting the prefix from the user.

Scatter recently implemented this feature, but currently requires users to know the prefix of the blockchain they're interacting with (e.g. https://github.com/GetScatter/ScatterDesktop/pull/19)

Looking forward to your feedback.

Thanks!

Reply:
Yes, that is your use case.  However it's not the only one.  If the intent is to provide an automated means to interact with an EOS.IO chain with an arbitrary CORE_SYMBOL, you also need to know the precision to perform transactions using the symbol.  Your code may not do that, but someone else may wish to write code that does.  Automating the full discovery required to successfully do that is perfectly reasonable.

Reply:
Good point. Will take a look and issue a pull request to the ``develop`` branch with the changes.

Reply:
New pull request with incorporated feedback submitted in https://github.com/EOSIO/eos/pull/5463

Regards